configure.ac

```diff
@@ -266,9 +266,32 @@ AC_CHECK_FUNCS(setns, [
 
 AC_SUBST(SINGULARITY_DEFINES)
 
-
-AC_CHECK_PROG([mksquashfs],[mksquashfs],[yes],[no])
-AM_CONDITIONAL([FOUND_MKSQUASHFS], [test "x$mksquashfs" = xyes])
+AC_MSG_CHECKING([--with-mksquashfs])
+AC_ARG_WITH([mksquashfs],
+            AS_HELP_STRING([--with-mksquashfs], [Root location of mksquashfs install])
+           )
+AS_IF([test "x$with_mksquashfs" != "x"],
+      [
+       AC_CHECK_PROG([MKSQUASHFS_CHECK],[mksquashfs],[yes],[no],[$with_mksquashfs/bin])
```

Comment:
By default `mksquashfs` isn't installed into a `bin` subdirectory, so maybe also check in `$with_mksquashfs/` directly? See https://github.com/easybuilders/easybuild-easyconfigs/pull/6287/files/4d84fcea3940a4c6afbd45722a590f83168801c7#diff-cefe1a16fe4b9a2e7bcdec604d37482e

Reply:
I'll need to build it ... but don't you do something like:

`./configure --prefix=/some/path`

Where `/some/path` is the root? Then `--bindir` is `/some/path/bin`?

Reply:
There's no `configure` script, only a `Makefile`, you can control where it gets installed via `make install INSTALL_DIR=...`, cfr. INSTALL_DIR

Reply:
Ahh... it's one of those. Fun!

I have to keep the assumption there's going to be a `bin/`.  Generally, the `with` options that expect a directory are the _root_ (`--prefix`) where it was installed, and assuming a autoconf configuration, there will be a `$prefix/bin`.

If someone else doesn't like it they can put in a PR to change it, or @GodloveD can change it... but, for me... I can't make myself do it.

Dave: If you think it would be good to change... the best way would probably be to force the entire path to the `mksquashfs` binary when the `--with-mksquashfs` option is used.

Reply:
That last idea actually makes more sense to me...

Reply:
> That last idea actually makes more sense to me...

Yeah... but it breaks how the `--with-*` options are expected (and it totally doesn't make more sense to me after years of putting the "root" location in when using it =) ).

We aren't doing anything fancy so it would probably work fine, but IIRC, some of the macros expect the given directory there to be the *root* location (i.e. the `--prefix` value).